package/standalone_file_logger/standalone_file_logger/src/rotating_logger.cc

```diff
@@ -189,9 +189,16 @@ double RotatingLogger::get_time_passed()
 void RotatingLogger::frame_handler(const uint8_t *data, size_t size)
 {
   // TODO: keep track of total data bytes: enforce a limit/warning in .h
+  if (_queue_bytes + size > MAX_QUEUE_SIZE) {
+    log_msg(LOG_WARNING,
+            std::string("Internal queue full, dropping bytes: ") + std::to_string(size));
```

Comment:
This may need a `.c_str()` call to work correctly with `log_msg`.

Reply:
log_msg itself is doing a `.c_str`
i also triggered it with maxqueue_size = 0, and it seems to print

Reply:
👍